internal/jobtracker/jobtracker.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/canonical/jimm/v3/internal/errors"
 )
 
+const minimumRefreshIntervalseconds = 5
```

Comment:
You can opt to make this `5*time.Second` and then below, since timeDuration implements the `Stringer` interface just use it in the error message with `%s`.